deploy/crds/dynatrace.com_oneagentapms_crd.yaml

```diff
@@ -1,18 +1,8 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Nope, nope, nope. Go back to v1beta1, it will fail otherwise on <K8s 1.16. Downgrade the Operator SDK if you have to.